README.md

```diff
@@ -186,6 +187,10 @@ You can then select the type of iOS device you'd like to use as a simulator
 
 ```bash
 npm run android-live-reload
+
+# If any errors occur, set the following environment variables and try again, or run the app using Android Studio
+export JAVA_HOME=/path/to/android-studio/jbr
+export ANDROID_SDK_ROOT=/path/to/Android/Sdk
```

Comment:
Not necessarily in scope for this PR, but I also encountered the following error when trying to run the Android server

```
✖ Creating capacitor.config.json in android/app/src/main/assets - failed!
✖ copy android - failed!
[error] Error: ENOENT: no such file or directory, open
        '/Users/ynda/code/github.com/salemlf/hakubun/android/app/src/main/assets/capacitor.config.json'
```

Creating the `assets` directory manually before re-running fixed that error, then I got onto the next error, which your suggestion would presumably fix

```
[error] native-run failed with error

        ERR_SDK_NOT_FOUND: No valid Android SDK root found.
```

Reply:
I also ran into this issue, so I added a `mkdir` inside the `android-live-reload` script in `package.json`, which fixed the issue for me.
Not sure if the file is meant to have compatibility with windows though, if so, I'll need to think of something else.